sync/fetch_queue.go

```diff
@@ -103,9 +112,9 @@ func (fq *fetchQueue) addToPending(ids []types.Hash32) []chan bool {
 	for _, id := range ids {
 		ch := make(chan bool, 1)
 		deps = append(deps, ch)
-		if _, ok := fq.pending[id]; !ok {
-			idsToAdd = append(idsToAdd, id)
-		}
+		//if _, ok := fq.pending[id]; !ok {
+		idsToAdd = append(idsToAdd, id)
+		//}
```

Comment:
Not sure why the conditional was removed here...

It seems that without the conditional `idsToAdd` is an exact copy of `ids`.

If this change was accidental, let's revert it. Otherwise, let's remove the commented out code and also the `idsToAdd` variable and use `ids` instead.